### ISSUE TITLE
Outposts ...

### ISSUE THREAD

**Issue by keybounce** [doublespeakgames/adarkroom#574]
So abandoned cities, when you clear them out, become outposts if you leave things behind. This is good -- it means you can leave extra meat behind, and get a water resupply point -- useful for getting to the far points of the map.

But ... I just checked out a battle field. It's been sitting there for who knows how long. Does it become a safe outpost when I leave it intact for later harvesting? 

No. It becomes a faded F that I can not go back to. Argh.

**Comment by keybounce** [comment]
In the same view: I'd like to be able to make outposts, or at least add stuff to an existing outpost. Right now it's hard -- I have to be full, and I can only leave as much extra meat as I can pick up of other stuff, so frequently the outposts become "water only" as if they are used for food they are emptied and lost.

**Comment by keybounce** [comment]
So the outposts actually fail the test of doing what a player expects.

What I expected: Each outpost has a fixed inventory.
What I saw: Whenever I visited an outpost, it went "ghosted" and was no longer usable.

But: It turns out that they come back on your next trip, and checking the source code makes it clear that each visit they just generate some random meat, and have no concept of a persistent inventory.

**Comment by AndySky21** [comment]
You really think that in a world where everything has been literally scavenged up to the bones, you leave a supply of food, ammo and weapons in an outpost and when returning you'll find everything sitting there to wait for you?
You're lucky to find something there (and just because otherwise it would be impossible to play).

**Comment by gitupofftheflooranddosomework** [comment]
I agree with @AndySky21, this should be closed, however perhaps reopened as a feature request (in such a manor that you could leave specific things at specific outposts however there is no guarantee that they wouldnt be stolen)
